client/tsconfig.json

```diff
@@ -11,7 +11,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react"
+    "jsx": "react",
+    "jsxImportSource": "@emotion/react"
```

Comment:
👍🏽

Reply:
Actually this compilerOption was causing errors when running tsc. I removed it and it seemed to work fine and styling is unchanged.